hack/restructure_file.py

```diff
@@ -1,56 +1,54 @@
 import json
 import sys
 
-filename= sys.argv[1]
-savefile= sys.argv[2]
-printfile= sys.argv[3]
+filename = sys.argv[1]
+savefile = sys.argv[2]
+printfile = sys.argv[3]
 
 
-def has_body(list1):
-    message=""
-    if list1["body"]:
-        message="True"
+def has_body(item1):
```

Comment:
I think this, and the one bellow, can be called just `item`, there is no need to enumerate them, they are different functions and the scope will never think of them as the same `item`.